src/others/monkey/lv_monkey_private.h

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file lv_monkey_private.h
+ *
+ */
+
+#ifndef LV_MONKEY_PRIVATE_H
+#define LV_MONKEY_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_monkey.h"
+
+#if LV_USE_MONKEY != 0
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+struct lv_monkey_config_t {
+    /**< Input device type*/
+    lv_indev_type_t type;
+
+    /**< Monkey execution period*/
+    struct {
+        //! @cond Doxygen_Suppress
```

Comment:
Why do we need this Doxygen comment?

Reply:
It got copy-pasted from the public header by my refactoring automation. I can remove it.